capstone/src/test/java/DeleteFlagsTest.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.*;
+import java.util.Date;
+
+public class DeleteFlagsTest {
+    @Test
+    public void testIsTooOld1() throws Throwable {
+        Date date1 = new Date(2020, 1, 1);
+        Date date2 = new Date(2020, 6, 1);
+        DeleteFlags underTest= new DeleteFlags();
```

Comment:
Run a linter or better yet an autoformatter on all files, not just the one I commented on earlier.

Reply:
Fixed in newer pull request.